invenio_records_files/config.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2019 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio-Records-Files configuration."""
+
+
+RECORDS_FILES_ENDPOINT_SUFFIX = 'files'
+"""/records/<pid>/<RECORDS_FILES_ENDPOINT_SUFFIX>."""
+
+RECORDS_FILES_REST_ENDPOINTS = {}
+"""REST endpoints which will have files mounted.
```

Comment:
Do we "mount" endpoints?

`REST endpoints for which the /files path (OR RESOURCE? OR ROUTE?) will be added`